tpot/tpot.py

```diff
@@ -261,16 +267,29 @@ def pareto_eq(ind1, ind2):
 
             self.hof = tools.ParetoFront(similar=pareto_eq)
 
-            stats = tools.Statistics(lambda ind: ind.fitness.values[1])
-            stats.register('Minimum score', np.min)
-            stats.register('Average score', np.mean)
-            stats.register('Maximum score', np.max)
+            # stats = tools.Statistics(lambda ind: ind.fitness.values[1])
+            # stats.register('Minimum score', np.min)
+            # stats.register('Average score', np.mean)
+            # stats.register('Maximum score', np.max)
 
             verbose = (self.verbosity == 2)
 
-            pop, _ = algorithms.eaSimple(population=pop, toolbox=self._toolbox, cxpb=self.crossover_rate,
+            # Start the progress bar
+            num_evaluations = self.population_size * (self.generations + 1)
+            self.pbar = tqdm(total=num_evaluations, unit='pipeline',
+                             disable=(not verbose), desc='GP Progress')
+
+            with warnings.catch_warnings():
+                # Ignore warnings
+                warnings.simplefilter('ignore', category=UserWarning)
```

Comment:
Some operators (like FastICA) can be noisy during a call to fit().

SKLearn would say that `tol` was too low for convergence, but I assumed the GP would just work that out and there wasn't any need to actually display those types of errors, especially considering how the user has no direct input when it comes to those evolve-able parameters and they mess with the progress bar.

Maybe tol just need to have a minimum allowed value for FastICA. I believe I've seen other operators have similar issues, though.